src/app/redux/alertAction.js

```diff
@@ -0,0 +1,44 @@
+import { SHOW_ALERT } from '../features/Chanels/redux/actionsTypes';
+
+export const showAlertAction = (alert) => ({
+  type: SHOW_ALERT,
+  payload: alert,
+});
+
+function showAlert(alertState) {
+  return async (dispatch) => {
+    dispatch(showAlertAction(alertState));
+    setTimeout(() => {
+      dispatch(
+        showAlertAction({
```

Comment:
add another action for hiding an alert. You can name it **hideAlert** or **resetAlertState**. In reducer, you'll have to just return the initialState of alert redcer.